SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-only

kconfig is bad, it uses gperf :(

--- Makefile	2021-07-20 13:51:59.750925575 +1000
+++ Makefile	2021-07-20 21:11:03.043389176 +1000
@@ -440,7 +440,6 @@
 
 config %config: scripts_basic outputmakefile FORCE
 	$(Q)mkdir -p include/linux
-	$(Q)$(MAKE) $(build)=scripts/kconfig $@
 	$(Q)$(MAKE) -C $(srctree) KBUILD_SRC= .kernelrelease
 
 else
@@ -453,6 +452,6 @@
 	$(Q)$(MAKE) $(build)=$(@)
 
-scripts_basic: include/linux/autoconf.h
+scripts_basic:
 
 # Objects we will link into vmlinux / subdirs we need to visit
 init-y		:= init/
